repose-aggregator/core/repose-core-api/src/main/java/org/openrepose/commons/config/manager/UpdateFailedException.java

```diff
@@ -20,6 +20,10 @@
 package org.openrepose.commons.config.manager;
 
 public class UpdateFailedException extends Exception {
+    public UpdateFailedException(String message) {
```

Comment:
Why was this added? I'm not seeing the usage (could be i'm losing it in everything else).

Reply:
It is used here:
> https://github.com/rackerlabs/repose/blob/688b55855bd5b4f0eff574858629ca34510c03d8/repose-aggregator/components/filters/regex-rbac-filter/src/main/scala/org/openrepose/filters/regexrbac/RegexRbacFilter.scala#L166